Cargo.toml

```diff
@@ -13,6 +13,7 @@ description = "Wireguard generic netlink packet definitions"
 [dependencies]
 anyhow = "1.0.42"
 byteorder = "1.4.3"
+getrandom = "0.2.10"
```

Comment:
The dependencies only used by examples or tests should be added as dev-dependencies instead.

Reply:
Right 👍 Updated.